source/localizable/addons-and-dependencies/managing-dependencies.md

```diff
@@ -109,7 +111,11 @@ All style assets added this way will be concatenated and output as `/assets/vend
 
 ### Other Assets
 
-All other assets like images or fonts can also be added via `import()`.
+All assets located in the `public` folder will be copied as is to the `dist/` folder.
```

Comment:
Even though it's covered on this page, a new user skimming the documentation may not be clear about what the `dist/` folder is. Maybe this should mention that `dist/` is the final output directory. Also, for consistency, we should probably add a trailing slash to `public/`.